File: JAVApb5/src/main/java/concurs/domain/AngajatOficiu.java
package concurs.domain;

import java.util.Objects;

public class AngajatOficiu  extends Entity<Long>{
    private String username;
    private String parola;

    public AngajatOficiu(String username, String parola) {
        this.username = username;
        this.parola = parola;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getParola() {
        return parola;
    }

    public void setParola(String parola) {
        this.parola = parola;
    }

    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;
        AngajatOficiu that = (AngajatOficiu) o;
        return Objects.equals(username, that.username) &&
                Objects.equals(parola, that.parola);
    }

    @Override
    public int hashCode() {
        return Objects.hash(username, parola);
    }

    @Override
    public String toString() {
        return "AngajatOficiu{" +
                "username='" + username + '\'' +
                ", parola='" + parola + '\'' +
                '}';
    }
}

File: CSharp/CSharp/CSharp/domain/AngajatOficiu.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CSharp.domain
{
    class AngajatOficiu:Entity<long>
    {
        private String Username { get; set; }
        private String Parola { get; set; }

        private AngajatOficiu(String Username, String Parola)
        {
            Username = Username;
            Parola = Parola;
        }
        public AngajatOficiu()
        { }
        public override bool Equals(object obj)
        {
            return obj is AngajatOficiu oficiu &&
                   Username == oficiu.Username &&
                   Parola == oficiu.Parola;
        }

        public override int GetHashCode()
        {
            int hashCode = 326650367;
            hashCode = hashCode * -1521134295 + EqualityComparer<string>.Default.GetHashCode(Username);
            hashCode = hashCode * -1521134295 + EqualityComparer<string>.Default.GetHashCode(Parola);
            return hashCode;
        }

        public override string ToString()
        {
            return base.ToString();
        }
    }
}
